src/SubmitPage/AvailabilityTags.js

```diff
@@ -13,39 +14,61 @@ class AvailabilityTags extends Component {
     super(props);
 
     this.state = {
-      tags: {
-        medecines: false,
-        groceries: false,
-        vegetables: false,
-        meat_and_dairy: false,
-        personal_and_homecare: false,
-      },
+      tags: [
+        { tagName: "medecines", checked: false, translationKey: "medecines" },
```

Comment:
Quick note: It's "medicines"

Reply:
Yeah. these are dummy values. i'll update them once all the logic related changes are done